src/content/software/guppy.md

```diff
@@ -0,0 +1,8 @@
+---
+name: GuPPy
+description: Guided Photometry Analysis in Python, a free and open-source fiber photometry data analysis tool. We contribute to this open-source project that aims to standardize and simplify fiber photometry workflows.
```

Comment:
Can you please add some more to this? Mention that this is in collaboration with the Lerner Lab, and talk briefly about what guppy does and how we are helping with development. 3-5 sentences